networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/nsxv3_policy.py

```diff
@@ -622,6 +622,62 @@ def get_revision(self, resource_container, resource_id):
         else:
             return -1
 
+    def get_orphan_ipsets(self):
```

Comment:
Can we have a test case / are we sure that this code doesn't delete anything in-use?